tools/wltests/test_series

```diff
@@ -786,12 +786,25 @@ match_sha1() {
 	COMMIT_SHA1=$1
 
 	c_info "Current kernel: "
-	CURRENT=$($ADB shell 'uname -a')
-	c_info "   $CURRENT"
+	c_info "   $($ADB shell 'uname -a')"
 
-	[[ $CURRENT = *$COMMIT_SHA1* ]] || return $EAGAIN
+	CURRENT=$($ADB shell 'uname -r | awk -F "-" "{print \$NF}"')
+	CURRENT=${CURRENT:1}
```

Comment:
This also can be folded in the previous command, thus:
```python
CURRENT=$($ADB shell 'uname -r' | awk -F "-g" "{print \$NF}"')
```

Reply:
Good.